core/src/main/scala/tech/beshu/ror/accesscontrol/factory/decoders/rules/IndicesRulesDecoders.scala

```diff
@@ -18,17 +18,19 @@ package tech.beshu.ror.accesscontrol.factory.decoders.rules
 
 import cats.data.NonEmptySet
```

Comment:
I don't know why decoders for Indices, Repositories, and Snapshots rules are in one file. Can we refactor it?